configs/reconciler.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 db:
-  driver: postgres
+  driver: sqlite
```

Comment:
Not sure if this will work.. on MAC I'm getting failiures that SQLite support isn't available in the default binaries

Reply:
Shouldn't the `reconciler.db` be removed from being tracked? 
It works if you change the `sqlite.file` entry.